newVersion.py

```diff
@@ -544,6 +544,19 @@ def add_common_stage_arguments(parser, new_mobile=True):
     parser.add_argument('-t', '--title', dest='title', help='Release title')
     parser.add_argument('--notes', dest='notes', help='Release notes')
 
+# We always want to publish our packages using Node 16 (>= 16.11), so check for that.
+# This insures that our package-lock.json files are conistent for npm.
+def check_node_version():
+    print("Verifying that node version is 16.x, with minimum of 16.11.")
+    results = subprocess.check_output(['node', '--version'], encoding='UTF-8')
+    match = re.search('^v16\\.([0-9]+)\\.', results)
+    if not match:
+        raise Exception("Error: Node 16.x required. You have " + results.rstrip('\n') + ".")
```

Comment:
I think this error message should be the same as the one on line 558.

Reply:
I joined the two checks into one, using the longer error

Reply:
Looks good